packages/salesforcedx-vscode-core/src/orgPicker/orgList.ts

```diff
@@ -43,7 +43,7 @@ export class OrgList implements vscode.Disposable {
   }
 
   public displayDefaultUsername(defaultUsernameorAlias?: string) {
-    if (!defaultUsernameorAlias) {
+    if (defaultUsernameorAlias) {
```

Comment:
Not sure why this worked the way it did before, but flipping this now has the org list showing up in command palette, shows the default org in the bottom menu bar, and appears to successfully be running "Set a default org" command.

Reply:
B/c somebody screwed up removing the deprecated function :mybad: 
`  if (!isNullOrUndefined(defaultUsernameorAlias)) {` was the original setting.

Your change totally makes sense. 👍